UVAtlas/isochart/isochartengine.cpp

```diff
@@ -37,40 +37,50 @@ void IIsochartEngine::ReleaseIsochartEngine(
 
 CIsochartEngine::CIsochartEngine() :
     m_state(ISOCHART_ST_UNINITILAIZED),
+#ifdef WIN32
     m_hMutex(nullptr),
+#endif
     m_dwOptions(_OPTION_ISOCHART_DEFAULT)
 {
 }
 
 CIsochartEngine::~CIsochartEngine()
 {
-    // if other thread is calling public method of CIsochartEngine this time, 
+    // if other thread is calling public method of CIsochartEngine this time,
     // Free will return with "busy". So loop until free successfully.
     while (FAILED(Free()))
     {
+#ifdef WIN32
         SwitchToThread();
```

Comment:
If you're willing to take a C++11 STL dependency, `std::this_thread::yield()` would cut down on this ifdef.

Reply:
I like that better than using pthreads...